test/GaslessV3.js

```diff
@@ -124,6 +126,7 @@ describe('Generic Contract Functions', function () {
         owner = getSigner(0);
         relayer = getSigner(1);
         main = await deployContract();
+        await main.deployed();
```

Comment:
let's add this inside `deployContract` function itself. was the code breaking without this?

Reply:
check this

Reply:
let's try this again